OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverShroudSelectorLogic.cs

```diff
@@ -97,13 +97,17 @@ public ObserverShroudSelectorLogic(Widget widget, ModData modData, World world,
 				.OrderBy(g => g.Key);
 
 			var noTeams = teams.Count() == 1;
+			var totalPlayers = 0;
 			foreach (var t in teams)
 			{
+				totalPlayers += t.Count();
 				var label = noTeams ? "Players" : t.Key == 0 ? "No Team" : $"Team {t.Key}";
 				groups.Add(label, t);
 			}
 
+			var shroudSelectorDisabled = limitViews && totalPlayers < 2;
```

Comment:
Why only do this for missions? It makes sense to disable this for other game types as well when there is only one player.

Reply:
Other game types show the all players / disable shroud options, so always have at least 3 items.

Reply:
I see, makes sense.